app/src/main/kotlin/com/proxerme/app/helper/MaterialDrawerHelper.kt

```diff
@@ -35,22 +36,42 @@ class MaterialDrawerHelper(context: Activity, toolbar: Toolbar,
 
     private val header: AccountHeader
     private val drawer: Drawer
+    //Only available if the device is a tablet
```

Comment:
I think `miniDrawer` and `crossfader` being nullable implies this comment so it is not necessarily needed.